tests/cxx/using_overload.cc

```diff
@@ -0,0 +1,30 @@
+//===--- using_overload.cc - test input file for iwyu ---------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
```

Comment:
I agree.  I'll do this for the tests I've added. At some point the existing tests will need to be updated.